spec/string_spec.rb

```diff
@@ -29,4 +31,35 @@
     end
   end
 
+  it "replaces spaces with underscores" do
+    Benchmark.ips do |x|
+      str = Random.new.bytes(1024)
```

Comment:
Hmmmm. How would you imagine using Faker? There's not a built in space or underscore helper I imagine. Build your own "Faker" class?